azurelinuxagent/common/protocol/wire.py

```diff
@@ -1137,7 +1137,9 @@ def report_event(self, event_list):
                 buf[event.providerId] = ""
             event_str = event_to_v1(event)
             if len(event_str) >= 63 * 1024:
-                logger.warn("Single event too large: {0}", event_str[300:])
+                details_of_event = [ustr(x.name) + ":" + ustr(x.value) for x in event.parameters if x.name in
+                                    ["Name", "Version", "Operation", "OperationSuccess"]]
+                logger.warn("Single event too large: {0}".format(str(details_of_event)))
```

Comment:
Main change - fixing the log for large size messages.

Reply:
we need a corresponding unit test in tests/protocol/test_wire.py

Reply:
Let's add the limit we are checking against to the warning to facilitate debugging